### DISCUSSION TITLE
Hola

### DISCUSSION THREAD

**Discussion by DavidB26** [Launch-X-Latam/MisionFrontEnd#434] [Team Posts]
Hola, recién me acaban de dar mis datos de acceso para las clases, pero no puedo registrarme en el grupo de teams, se tiene que hacer algo más?

**Comment by infectedInnovaccion** [comment]
Nop, esperar

**Comment by T0nyStrong777** [comment]
ya has podido entrar DavidB26?

**Comment by Amg79-2022** [comment]
Hola, si ya pude acceder, muchas gracias 🙏

Enviado desde Yahoo Mail para Android 
 
  El vie, 7 de oct. de 2022 a la(s) 8:50 p. m., ***@***.***> escribió:   


ya has podido entrar DavidB26?

**Comment by T0nyStrong777** [comment]
Muy bien a seguir estudiando, que tengan un gran fin de semana

**Comment by DavidB26** [comment]
No, no me deja ingresar, me sigue saliendo error de contraseña.

**Comment by gumodi35** [comment]
Puede que esté intentando copiar y pegar si es eso... Mejor escribirla.
Si no es ese el problema, borra tu historial de navegación, y nuevamente ingresa, probablemente eso también sirva .

**Comment by T0nyStrong777** [comment]
Lo que yo he hecho es abrir, el navegador chrome en el perfil de invitado, abrir el navegador de office 365 login, le puse usuario y contraseña me dio resultado
